apps/server/src/shared/domain/entity/lesson.entity.ts

```diff
@@ -67,7 +68,7 @@ export type ComponentProperties = {
 	_id?: string;
 	title: string;
 	hidden: boolean;
-	user?: EntityId;
+	user?: ObjectId;
```

Comment:
Is this change really needed? The entityId is the same type like string and this is what i expect if we do not load a full entity.

Reply:
@WojciechGrancow

Reply:
In the end all the references are objectId and the ORM has a detection method for it. But if i remember correct than in one of the ORM release changelogs maybe end last year, or begin of this year it exists a note that something is changed. 
For the most places we work directly with adding the entity and therefore it is no problem. 

I must check it. (todo: @CeEv )

Reply:
Since there is no LessonDO, I find this change dangerous. If the repo needs to use an objectId for a query it can create one out of the string, but here in the entity it should remain an EntityId, as is the case everywhere else.
The MikroORM is mapping this automatically.

Please revert this change.

Reply:
Please let the EntityId on this place. In repo layer by building the query you add with this PR already the new ObjectId(<id>) this should be enough that it is work.

Reply:
done